faqs.rst

```diff
@@ -115,6 +115,13 @@ and throughput:
   `Flux python interface <https://flux-framework.readthedocs.io/projects/flux-core/en/latest/python/index.html>`_.  The
   `flux-mini <https://github.com/flux-framework/flux-core/blob/master/src/cmd/flux-mini.py>`_
   command itself is a python program that can be a useful reference.
+- If jobs produce much output on stdout or stderr, use the
+  :core:man1:`flux-mini` ``--output`` and ``--error`` options to redirect them
```

Comment:
`--output` redirects both stdout and stderr, so it is probably sufficient to just include that option in this short faq answer. (O/w, users might think they have to supply both to redirect both)